src/Ledger/Foreign/HSLedger.agda

```diff
@@ -156,11 +156,12 @@ HsGovParams : GovParams
 HsGovParams = record
   { Implementation
```

Comment:
Oh, I wasn't familiar with this syntax.  So, does this open up the `Implementation` module inside the record definition?  ...cool!